sst/config/config.py

```diff
@@ -0,0 +1,62 @@
+import sst
+
+DEBUG_L1 = 0
+DEBUG_MEM = 0
+DEBUG_LEVEL = 0
+
+# Define the simulation components
+cpu = sst.Component("core", "sstsimeng.simengcore")
+cpu.addParams({
+    "config_path": "/home/rahat/asimov/SimEng/configs/a64fx.yaml",
```

Comment:
change config path to a64fx-sst.config.py

Reply:
Left unresolved just in case we change location. We should also change some of these config params to purposely fail to ensure a new user is running what they mean to.